cmd/createtree/main_test.go

```diff
@@ -218,6 +232,9 @@ func (s *fakeAdminServer) CreateTree(ctx context.Context, req *trillian.CreateTr
 		return nil, s.err
 	}
 	resp := *req.Tree
+	if req.KeySpec != nil && s.generatedKey != nil {
```

Comment:
I've made it panic in this case. While I know panic is to be avoided, it seems the best option here as I don't want tests to accidentally interpret this mistake in test setup as an expected error (in those test cases where `fakeAdminServer.err != nil`).